qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/difference_table_widget/difference_table_widget_view.py

```diff
@@ -205,9 +205,13 @@ def on_item_changed(self):
         if not self._updating:
             pass
 
-    def on_cell_changed(self, _row, _col):
+    def on_cell_changed(self, row, column):
         if not self._updating:
-            self._on_table_data_changed(_row, _col)
+            if not (isinstance(self.sender(), QtWidgets.QTableWidget) or self.sender().pos().isNull()):
```

Comment:
Sorry, I know this is a post-gatekeeper comment, but I thought I'd just mention this style option in case you hadn't thought of it and might find it interesting for the future (obviously no need to change anything here!). Depending on preference, if you wanted to avoid the nested `if` statements and keep the indentation simpler in this method then I think you could write this as follows:

```
def on_cell_changed(self, row, column):
    if self._updating:
        return
		
    if not (isinstance(self.sender(), QtWidgets.QTableWidget) or self.sender().pos().isNull()):
        pos_index = self.pairing_table.indexAt(self.sender().pos())
        row = pos_index.row()
        column = pos_index.column()

    self._on_table_data_changed(row, column)
```

We use this style quite a bit in LSS, so I just thought I'd share the thought 🙂

Reply:
Thanks for the suggestion. I agree it looks more neat, I'll take into account.